src/test/resources/regressions/examples/tutorial-examples/basic-annotations.gobra

```diff
@@ -4,34 +4,31 @@
 package tutorial
 
 requires 0 <= n // precondition
-ensures sum == n * (n+1)/2 // postcondition
-decreases
+ensures  sum == n * (n+1)/2 // postcondition
 func sum(n int) (sum int) {
-	sum := 0
+  sum = 0
 
-	invariant 0 <= i && i <= n + 1 
-	invariant sum == i * (i-1)/2
-	decreases n - i
-	for i := 0; i <= n; i++ {
-			sum += i
-	}
-	return sum
+  invariant 0 <= i && i <= n + 1 
+  invariant sum == i * (i-1)/2
+  for i := 0; i <= n; i++ {
+    sum += i
+  }
+  return sum
 }
 
-ensures res == (n % 2 == 0)
 decreases
+ensures res == (n % 2 == 0)
 pure func isEven(n int) (res bool) {
-	return n % 2 == 0
+  return n % 2 == 0
 }
 
 ensures isEven(n) ==> res == n / 2
 ensures !isEven(n) ==> res == n / 2 + 1
-decreases
 func halfRoundedUp(n int) (res int) {
```

Comment:
is there a reason why we are changing the indentation here? I'd rather keep tabs over spaces, as this is what gofmt enforces in .go files (same applies to all other files)

Reply:
I changed it to spaces to allow seamless copy-pasting from the `tutorial-examples` to `tutorial.md` but will revert this change. Is it fine to indent using tabs in `tutorial.md`?

Reply:
yup, fine by me!